dask/dataframe/io/tests/test_parquet.py

```diff
@@ -938,6 +938,60 @@ def test_to_parquet_default_writes_nulls(tmpdir):
     assert table[1].null_count == 2
 
 
+def test_to_parquet_consistent_schema_partition_pyarrow(tmpdir, engine):
+    check_fastparquet()
+    check_pyarrow()
+    fn = str(tmpdir.join("dask.pa"))
+
+    arrays = np.array([0, 1, 2, 4])
+    strings = np.array(["a", "b", "c", "d"])
+
+    df = pd.DataFrame([0, 0, 1, 1], columns=["partition_column"])
+    df["arrays"] = pd.Series(arrays)
+    df["strings"] = pd.Series(strings)
+
+    ddf = dd.from_pandas(df, npartitions=2)
+    ddf.to_parquet(fn, engine=engine, partition_on=["partition_column"])
+    ddf2 = dd.read_parquet(fn, engine=engine)
+
+    assert np.array_equal(ddf2["arrays"], arrays)
+    assert np.array_equal(ddf2["strings"], strings)
+    assert np.array_equal(ddf2["partition_column"], df["partition_column"])
```

Comment:
This test doesn't seem to use the `schema=` keyword?

Reply:
As I mention above, this test does not appear to be directly related to the code change. Have pinged @mmccarty to see what his intention was with this test. I hate to let a good test go to waste but I'm happy to delete this one.